slf4j-site/pom.xml

```diff
@@ -20,7 +20,7 @@
   <build>
     <resources>
       <resource>
-        <directory>src/site/pages</directory>
+        <directory>${project.basedir}/src/site/pages</directory>
         <!--We're saving filtered html docs in a temporary folder-->
         <!--and telling the site plug in to get the docs there.-->
         <targetPath>../../../target/site</targetPath>
```

Comment:
In that location it would look in the wrong place.  The base directory is the module it is in.  In this case that info is outside of that module.  I can take at making that part of the build more maven friendly in another PR.

--- Original Message ---

From: "Matt Bishop" <notifications@github.com>
Sent: April 8, 2015 12:30 AM
To: "qos-ch/slf4j" <slf4j@noreply.github.com>
Cc: "Jeremy Landis" <jeremylandis@hotmail.com>
Subject: Re: [slf4j] Apply Standard Maven Usage (#107)

> @@ -20,7 +20,7 @@
>    <build>
>      <resources>
>        <resource>
> -        <directory>src/site/pages</directory>
> +        <directory>${project.basedir}/src/site/pages</directory>
>          <!--We're saving filtered html docs in a temporary folder-->
>          <!--and telling the site plug in to get the docs there.-->
>          <targetPath>../../../target/site</targetPath>

Why isn't this ${project.basedir}/target/site ?

---
Reply to this email directly or view it on GitHub:
https://github.com/qos-ch/slf4j/pull/107/files#r27942099